ci-operator/jobs/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master-presubmits.yaml

```diff
@@ -1,5 +1,109 @@
 presubmits:
   rh-ecosystem-edge/ci-artifacts:
+  - agent: kubernetes
+    always_run: true
```

Comment:
I don't think you want to `always_run: true` these `ci/prow/4.6-images`,
all the variants use the same image definition, only one `ci/prow/images` in the main `master` file is enough

Reply:
correct